post-build-dockerfile

```diff
@@ -11,15 +11,29 @@ source "$(dirname "$0")/config"
 APP="$1"; APP_VOLUME="$VOLUMES_ROOT/$APP"; IMAGE="dokku/$APP"
 VOLUMES=$(docker inspect --format='{{range $volume, $value := .Config.Volumes}}{{$volume}}{{"\n"}}{{end}}' "$IMAGE")
 
-for VOLUME in $VOLUMES; do
-  VOLUME_PATH="$APP_VOLUME$VOLUME"
-  VOLUME_DIR=$(dirname "$VOLUME_PATH")
-  if [[ -e $VOLUME_PATH ]]; then
+for VOLUME in ${VOLUMES}; do
+  # Add slashes before spaces
+  VOLUME=$(echo ${VOLUME} | awk '{gsub(/ /,"\\ ")}8')
+  VOLUME_PATH="${APP_VOLUME}${VOLUME}"
+  VOLUME_DIR=$(dirname "${VOLUME_PATH}")
+  if [[ -e ${VOLUME_PATH} ]]; then
     continue
   fi
-  mkdir -p "$VOLUME_DIR"
-  docker run --rm -v "$VOLUME_DIR:/target" "$IMAGE" cp -a "$VOLUME" /target
-  dokku docker-options:add "$APP" run,deploy "-v $VOLUME_PATH:$VOLUME"
+  dokku_log_info1 "VOLUME: Prepearing for: ${VOLUME}"
```

Comment:
dokku_log helps me to see more information about processes on server side at deploy.